warehouse/static/js/warehouse/utils/search-filter-toggle.js

```diff
@@ -0,0 +1,29 @@
+export default () => {
+  const togglePanelDisplay = (display, event) => {
+    event.preventDefault();
+    var elements = document.querySelectorAll(".-js-dark-overlay, .-js-filter-panel");
+    for (var el of elements) {
+      el.style.display = display;
```

Comment:
I understand that this is basically what the jQuery version does, but perhaps we could defer `el.style.display` part to a new CSS class (something like `.filter-panel--closed { display: none; }`)?